tests/test_connection.py

```diff
@@ -1,13 +1,17 @@
 import requests
+import os
+from dotenv import load_dotenv
 
 from sylveon._classes._login_helpers import (
     login_set_session_cookies,
     get_auth_token_init_gradescope_session,
 )
 
-#TODO create a test account
-TEST_EMAIL = "email"
-TEST_PASSWORD = "password"
+# load .env file
+load_dotenv()
+
+TEST_EMAIL = os.getenv("EMAIL")
```

Comment:
was creating a test account too complicated?
I feel like having each of us test using different accounts may result in unpredictable results (passing for one, but not for another) which may result in problems down the road

Reply:
it's not, we can always do that in the future (such as for CI). either way we should not have credentials lying around in code like this as there are bots that scrape login credentials.

Reply:
the only thing that we would change is the value in the .env file itself, we can just make sure everyone has the same credentials for the test account

Reply:
ok thats reasonable

Reply:
For CI we can also used Github secrets and store the test credentials there, so this seems good to me.